src/plugins/flexsearch.js

```diff
@@ -40,7 +61,7 @@ async function createFlexsearchIndexData() {
     .use(remarkExtractSections)
     .use(remarkHtml);
 
-  for (const filepath of await promisify(glob)('src/content/**/*.{md,mdx}')) {
+  for (const filepath of await promisify(glob)('src/content/**/api/*.{md,mdx}')) {
```

Comment:
@alexBinns will this remove the ability to search our guides?

Reply:
Good catch this was for testing. I'll revert this change.